### PR TITLE
Bump EKB to release-v1.4 

### PR DESCRIPTION
**Pull request by pierDipi**
As per title.

Signed-off-by: Pierangelo Di Pilato <pierdipi@redhat.com>

### CODE REVIEW

File: knative-operator/deploy/resources/knativekafka/channel/eventing-kafka-channel.yaml [hunk 1]
```diff
@@ -495,7 +494,6 @@ spec:
           args:
             - "-XX:+CrashOnOutOfMemoryError"
             - "-Dlogback.configurationFile=/etc/logging/config.xml"
-            - "-Dcom.redhat.fips=false"
```
Comment (pierDipi) [inline knative-operator/deploy/resources/knativekafka/channel/eventing-kafka-channel.yaml:498]: TODO: FIPS arguments is missing

**Comment by pierDipi** [comment]
Blocked by https://github.com/openshift/release/pull/30725 (due to image mirroring)

**Comment by pierDipi** [comment]
```

Unable to connect to the server: dial tcp: lookup api.ci-ocp-4-10-amd64-aws-us-east-1-hlwhz.hive.aws.ci.openshift.org on 172.30.0.10:53: no such host
```
/test 4.10-upstream-e2e-aws-ocp-410

**Review by aliok — APPROVED** [review]
/lgtm
/approve

File: olm-catalog/serverless-operator/project.yaml [hunk 2]
```diff
@@ -38,9 +38,9 @@ dependencies:
 
   eventing_kafka: 1.1.0
```
Comment (skonto) [inline olm-catalog/serverless-operator/project.yaml:39]: Should we remove this for 1.25?

Comment (pierDipi) [reply olm-catalog/serverless-operator/project.yaml:39]: QE is still using this for testing, let's keep it, it's only used for building the source image.

**Comment by pierDipi** [comment]
```
Error from server (NotFound): secrets "ca-key-pair" not found
error: failed to create secret Post "https://api.ci-ocp-4-10-amd64-aws-us-east-1-7mfhd.hive.aws.ci.openshift.org:6443/api/v1/namespaces/cert-manager/secrets?fieldManager=kubectl-create": dial tcp: lookup api.ci-ocp-4-10-amd64-aws-us-east-1-7mfhd.hive.aws.ci.openshift.org on 172.30.0.10:53: read udp 10.128.164.188:48580->172.30.0.10:53: i/o timeout
```
/retest

**Pull request merged**